cmd/crossbuild.go

```diff
@@ -132,21 +127,20 @@ func runCrossbuild() {
 		dockerS390XBuilderImage   = fmt.Sprintf("%s:%s-s390x", dockerBuilderImageName, goVersion)
 	)
 
+	var filteredPlatforms []string
```

Comment:
if you make it anchored, like in prometheus, it would be backwards compatible

Reply:
Except that would break the alias functionality of things like `linux/arm`. I've made it intentionally un-anchored so this would work.

Reply:
Can't we break it ?

Reply:
or "hardcode" the current aliases?

Reply:
But, that's the whole point of this. I don't want to hardcode these things. This PR should be backwards compatible, but I haven't tried all corner cases.